packages/help-center/src/components/help-center-contact-form.tsx

```diff
@@ -266,7 +267,7 @@ export const HelpCenterContactForm = ( props: HelpCenterContactFormProps ) => {
 	}
 
 	function handleCTA() {
-		if ( ! enableGPTResponse && ! showingSearchResults && ! wapuuFlow ) {
+		if ( ! enableGPTResponse && ! showingSearchResults && ! wapuuFlow && ! DIFMInProgress ) {
```

Comment:
I would have loved if we were able to just use `show-results`: `false`, but maybe this is more complex.

What do you think of generalizing this a little, allowing whoever wants to skip the recommended resources to simply be able to set the query param?
So, instead of `difm-in-progress`, maybe something like `skip-resources` or similar?